homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
```

Comment:
Room for improvement: the reason we like entitydescriptions is that we can create a generic class and then change the value it presents with entity descriptions. I would argue that in awair this isn't perfect, as `native_value` (and `available` for that matter) is directly checking what type of sensor it is. Instead, we could extend the awair entity_description with a value_fn and have the logic to fetch the data from the `self._air_data` in the entity description instead of the class. This would make it more extendable.